### ISSUE TITLE
Question: Can we build this repo in MonoDevelop under Ubuntu?

### ISSUE THREAD

**Issue by rahuldeo2047** [KeenSoftwareHouse/SpaceEngineers#586]
https://en.wikipedia.org/wiki/MonoDevelop

**Comment by mccorkle** [comment]
No. Even if you could build it in Ubuntu, the physics library (and other licensed libraries) are windows only, so you might have it compiled, but it would never launch. 

Before planets, we had the dedicated server running under Ubuntu via wine, but wine only supports 32 bit mode, and planets needs more memory than you can address in 32bit wine.

**Comment by kde99** [comment]
Not just the physics but also Direct X is windows only too. But you could build C hashtag on linux too. And yeah havoc is an other issue. I recommend you read Issue #9 .

**Comment by pwhelan** [comment]
WINE supports 64 bit and has for quite a while now.
